src/AppInstallerRepositoryCore/Microsoft/Schema/Checkpoint_1_0/CheckpointTable.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "CheckpointTable.h"
+#include "SQLiteStatementBuilder.h"
+
+namespace AppInstaller::Repository::Microsoft::Schema::Checkpoint_V1_0
+{
+    using namespace std::string_view_literals;
+    static constexpr std::string_view s_CheckpointTable_Table_Name = "Checkpoints"sv;
+    static constexpr std::string_view s_CheckpointTable_Name_Column = "Name";
+    static constexpr std::string_view s_CheckpointTable_WriteTime_Column = "WriteTime";
```

Comment:
This should be "CreationTime" unless you are planning to update it on every field write.